Library/Homebrew/cmd/bottle.rb

```diff
@@ -84,10 +87,15 @@ def print_filename(string, filename)
           result = true
 
           if ARGV.verbose?
+            strings_matches += 1
+            next if strings_matches > MAXIMUM_STRING_MATCHES
             print_filename string, file
             puts " #{Tty.gray}-->#{Tty.reset} match '#{match}' at offset #{Tty.em}0x#{offset}#{Tty.reset}"
           end
         end
+        if strings_matches > MAXIMUM_STRING_MATCHES
+          puts "Only the first #{MAXIMUM_STRING_MATCHES} matches were output"
+        end
       end
```

Comment:
I'd rather all the linked_libraries were always printed. The problem is only the huge number of string matches. I'd be open to making this filenames instead (but even then there may be huge numbers of results).